xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -114,7 +119,26 @@ private String exportHTML(XWikiContext context) throws XWikiException, IOExcepti
             name = context.getDoc().getFullName();
         }
 
-        Collection<DocumentReference> pageList = resolvePagesToExport(request.getParameterValues("pages"), context);
+        String[] checkedPages = request.getParameterValues("pages");
+        String[] uncheckedPages = null;
+        String checkedPagesRequestValue = request.get("checked-pages");
+        String uncheckedPagesRequestValue = request.get("unchecked-pages");
+        String otherPagesRequestValue = request.get("other-pages");
+
+        boolean otherPages = false;
+        if (!StringUtils.isEmpty(otherPagesRequestValue)) {
```

Comment:
From the javadoc `Boolean valueOf(String s)`:
> Returns a Boolean with a value represented by the specified string. The Boolean returned represents a true value if the string argument is not null and is equal, ignoring case, to the string "true".

So you might not need this line.

Reply:
Yes, you can assign directly ``Boolean.valueOf()``.